ci-operator/step-registry/ipi/conf/azurestack/ipi-conf-azurestack-ref.yaml

```diff
@@ -0,0 +1,36 @@
+ref:
+  as: ipi-conf-azurestack
+  from_image:
+   namespace: origin
+   name: centos
+   tag: '8'
+  commands: ipi-conf-azurestack-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  credentials:
+  - namespace: test-credentials
+    name: azurestack-cluster-secrets
+    mount_path: /var/run/azurestack-cluster-secrets
+  env:
+  - name: SIZE_VARIANT
+    default: ""
+    documentation: |-
+      The size of the cluster in one of our supported t-shirt values that is standard across all CI environments.
+
+      The sizes are:
+      * "" (default) - 4 vCPU, 16GB control plane nodes, default workers
+      * "compact" - 8 vCPU, 32GB control plane nodes, no workers
+      * "large" - 16 vCPU, 64GB+ control plane nodes, default workers, suitable for clusters up to 250 nodes
+      * "xlarge" - 32 vCPU, 128GB+ control plane nodes, default workers, suitable for clusters up to 1000 nodes
+
+      These sizes are roughly consistent across all cloud providers, but we may not be able to instantiate some sizes
+      in some regions or accounts due to quota issues.
+
+  - name: COMPUTE_NODE_TYPE
+    default: 'Standard_D4s_v3'
```

Comment:
The errors you're seeing now in CI: 
```
"Standard_D4s_v3": not found in region ppe3
``` 
are probably because the master_type and compute_type are being set explicitly in the install config with values that are not compatible with Azure Stack. To simplify this, you could simply remove setting these types in the install config, and use the installer defaults.